Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -306,10 +306,9 @@ extension CarPlayMapViewController: StyleManagerDelegate {
     }
     
     public func styleManager(_ styleManager: StyleManager, didApply style: Style) {
-        let styleURL = style.previewMapStyleURL
```

Comment:
Fixes #3224. There should be an ability to change `MapView` style URL similarly to iOS.